public/index.css

```diff
@@ -8,6 +8,26 @@
 	outline: none;
 }
 
+/* Firefox */
+* {
+	scrollbar-color: var(--theme-dim-light) transparent;
+}
+
+/* Webkit */
+
+::-webkit-scrollbar,
+::-webkit-scrollbar-track {
+	/* same as --theme-bg-gradient but without 'fixed' */
+	background: linear-gradient(180deg, var(--theme-bg-gradient-top), var(--theme-bg-gradient-top) var(--theme-navbar-height), var(--theme-bg-gradient-bottom));
```

Comment:
What does changing this from `transparent` fix? I’m seeing a slightly off scrollbar background, which I think wouldn’t be there if we were using `transparent`:

<img width="329" alt="image" src="https://user-images.githubusercontent.com/357379/167316483-1386bb8e-7c22-45c2-98be-7e4fc94f3a97.png">

But maybe that’s needed on other platforms?

Reply:
What happens is that the main scrollbar looks like this when it is as transparent:
![image](https://user-images.githubusercontent.com/61414485/167316765-e8470236-5a6d-4686-be17-807f9d2b1a3e.png)

I did some research but I didn't find a solution for this, my monitor has low color contrast so I didn't even notice the difference in color, nice catch!

`transparent` still works for the (Right/Left)Sidebar components though, we can override the `background` if we don't find a global solution for this.

Reply:
@delucis  I dont know if @Yan-Thomas  has fixed this before I had reviewed this, but I cant seem to see this trackbar any more, as a result Im pretty confident that this has conversation has been resolved 🙂